cloudwatch/cloudwatch_test.go

```diff
@@ -149,7 +149,7 @@ func (c mockCloudWatchClient) GetMetricData(input *cloudwatch.GetMetricDataInput
 	}
 
 	for i := 0; i < 10; i++ {
-		id := fmt.Sprintf("q{i}")
+		id := "q{i}"
```

Comment:
Ooops, I think I see what this was originally meant to be, but looks like this was a Python format string instead. Don't think it matters much, but maybe less confusing if it's this instead?
```suggestion
		id :=  fmt.Sprintf("q%d", i)
```